tools/scrimlet/softnpu-init.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Gateway ip is automatically configured based on the default route on your development machine
+# Can be overridden by setting GATEWAY_IP
+GATEWAY_IP=${GATEWAY_IP:=$(netstat -rn | grep default | awk -F ' ' '{print $2}')}
```

Comment:
If we're assuming IPv4 here, which I think we are given the `arp` call below, the `netstat` call needs to be changed to `netstat -rn -f inet`. Otherwise you'll get two entries and `GATEWAY_IP` will contain two addresses on two lines.

On my machine

```
ry@masaka:~$ netstat -rn | grep default | awk -F ' ' '{print $2}'
192.168.1.1
fe80::92ec:77ff:fe2e:7027
```

versus

```
ry@masaka:~$ netstat -rn -f inet | grep default | awk -F ' ' '{print $2}'
192.168.1.1
```

Reply:
Awesome, will add this change.